lib/import/admin.js

```diff
@@ -203,14 +190,10 @@ function import_admins(country_code, geojson, population, verbose) {
                            verbose);
   })
   .then(function(feature_collection) {
-    return save_features(feature_collection);
+    return save_admins(country_code, feature_collection);
   })
   .then(function(feature_collection) {
-    return save_topojson(
-      country_code,
-      feature_collection,
-      geojson,
-      verbose);
+    return save_topojson(country_code, feature_collection);
```

Comment:
I don't think these need to be sequenced this way any more. What do you think of:
```
.then(function(feature_collection) {
  return Promise.all([save_admins(country_code, feature_collection),
                      save_topojson(country_code, feature_collection)]);
})